msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -26,119 +26,111 @@ At a high level the steps you'll need to complete to send a proactive message ar
 
 * **What happened.** A clear indication of what happened to cause the notification.
-* **What was the result.** It should be clear what item/thing was updated to cause the notification.
+* **What was the result.** It must be clear what item or thing was updated to cause the notification.
 * **Who/what triggered it.** Who or what took action that caused the notification to be sent.
 * **What can users do in response.** Make it easy for your users to take actions based on your notifications.
-* **How can users opt out.** You need to provide a path for users to opt out of additional notifications.
+* **How can users opt out.** You must provide a path for users to opt out of additional notifications.
 
 ## Proactively install your app using Graph
 
 > [!Note]
 > Proactively installing apps using the Microsoft Graph is currently in beta.
 
-Occasionally it may be necessary to proactively message users that have not installed or interacted with your app previously. For example, you want to use the [company communicator](~/samples/app-templates.md#company-communicator) to send messages to your entire organization. For this scenario you can use the Graph API to proactively install your app for your users, then cache the necessary values from the `conversationUpdate` event your app will receive upon install.
+Occasionally it may be necessary to proactively message users that have not installed or interacted with your app previously. For example, you want to use the [company communicator](~/samples/app-templates.md#company-communicator) to send messages to your entire organization. For this scenario you can use the Graph API to proactively install your app for your users, then cache the necessary values from the `conversationUpdate` event your app receives upon installation.
 
-You can only install apps that are in your organizational app catalogue, or the Teams app store.
+You can only install apps that are in your organizational app catalog or the Teams app store.
 
-See [Install apps for users](/graph/teams-proactive-messaging) in the Graph documentation and [Proactive bot installation and messaging in Teams with Microsoft Graph](../../../graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md). There is also a [Microsoft .NET framework sample](https://github.com/microsoftgraph/contoso-airlines-teams-sample/blob/283523d45f5ce416111dfc34b8e49728b5012739/project/Models/GraphService.cs#L176)  on the GitHub platform.
+See [Install apps for users](/graph/teams-proactive-messaging) in the Graph documentation and [Proactive bot installation and messaging in Teams with Microsoft Graph](../../../graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md). There is also a [Microsoft .NET framework sample](https://github.com/microsoftgraph/contoso-airlines-teams-sample/blob/283523d45f5ce416111dfc34b8e49728b5012739/project/Models/GraphService.cs#L176) on the GitHub platform.
```

Comment:
[Install apps for users](https://docs.microsoft.com/en-us/graph/api/userteamwork-post-installedapps?view=graph-rest-1.0&tabs=http)

Reply:
Broken link

Reply:
Added the absolute path for **Install apps for users**.